js/views/startScreen.js

```diff
@@ -25,6 +25,10 @@ var StartScreen = React.createClass({
     this.handleResize();
   },
 
+  componentWillReceiveProps: function() {
```

Comment:
@mistermendez: Passing param to  handleResize() also will not work.  When you change the player size, then nextProps.contentTree.description and this.props.contentTree.description will be same. In this scenario handleResize() will not be called and description will be shown in more than 2 lines
<img width="1920" alt="screen shot 2016-12-09 at 10 09 19 am" src="https://cloud.githubusercontent.com/assets/16229782/21059585/f5394c46-bdf7-11e6-8854-3a2571cb8f26.png">